src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequest.java

```diff
@@ -23,11 +52,46 @@ public String getCompanyNumber() {
         return companyNumber;
     }
 
+    public Map<String, String> getAttachments() {
+        return attachments;
+    }
+
+    public String getReferenceNumber() {
+        return referenceNumber;
+    }
+
+    public String getCustomerEmail() {
+        return customerEmail;
+    }
+
+    public String getReason() {
+        return reason;
+    }
+
+    private Map<String, String> buildAttachmentsMap(String downloadPrefix,
+                                                    List<Attachment> attachments) {
+        Map<String, String> attachmentsMap = new HashMap<>();
+        for (Attachment attachment : attachments) {
+            String name = attachment.getName();
+            Links links = attachment.getLinks();
+            if(links != null) {
+                String downloadLink = String.format("%s%s", downloadPrefix,
+                        links.getLink(CoreLinkKeys.SELF));
```

Comment:
this should be the download link instead of self, and also should reference ObjectionLinkKeys, so use ObjectionLinkKeys.DOWNLOAD instead of SELF

Reply:
Ah alas if only there were one.

Reply:
Using ObjectionLinkKeys instead.